barcode/_version.py

```diff
@@ -0,0 +1,2 @@
+from __future__ import unicode_literals
+__version__ = '14.0'
```

Comment:
yea, either that or just make major minor versions of the plugin unrelated to anything. if we make big breaking changes, bump the major version up.  

either way